docs/ReleaseNewVersion.md

```diff
@@ -0,0 +1,39 @@
+# Releasing a new version
+
+This guide is to release a new version of the Code-checker plugin. Remember that when considering the version number to use, that
+ this project follows [Semantic Versioning](http://semver.org/), so bump the version number accordingly.
+
+Check the existing PRs and approve those that are applicable to this release.
+
+Create a standard PR with the version bump. Ensure the following have been updated:
+
+* The `CHANGE.md`, adding the PRs that have been approved and other existing changes.
```

Comment:
CHANGES (plural)?

Reply:
Changes better one at a time ;) You are right.